manifests/0000_12_cluster-kube-controller-manager-operator_06_deployment.yaml

```diff
@@ -31,6 +31,9 @@ spec:
         env:
         - name: IMAGE
           value: quay.io/openshift/origin-hyperkube:v4.0
+        - name: OPERATOR_IMAGE
+          value: docker.io/openshift/origin-cluster-kube-apiserver-operator:v4.0
```

Comment:
@deads2k this is not right
```
container create failed: container_linux.go:336: starting container process caused "exec: \"cluster-kube-controller-manager-operator\": executable file not found in $PATH"
```
should be `docker.io/openshift/origin-cluster-kube-controller-manager-operator:v4.0`